packages/sdk/app-framework/src/plugins/PluginHost/PluginContext.tsx

```diff
@@ -43,19 +44,12 @@ export type PluginContext = {
   setPlugin: (id: string, enabled: boolean) => void;
 };
 
-const PluginContext: Context<PluginContext> = createContext<PluginContext>({
-  ready: false,
-  core: [],
-  enabled: [],
-  plugins: [],
-  available: [],
-  setPlugin: () => {},
-});
+const PluginContext = createContext<PluginContext | undefined>(undefined);
```

Comment:
it's a strong pattern we have elsewhere and i think it's more correct since the provider is required (so the hook will fail if the context isn't set).